common-content/en/module/js1/arrow-functions/index.md

```diff
@@ -0,0 +1,60 @@
++++
+title = '➡️ Arrow functions'
+
+time = 5
+facilitation = false
+emoji= '➡️'
+[objectives]
+    1='Write an arrow function'
+[build]
+  render = 'never'
+  list = 'local'
+  publishResources = false
+
++++
+
+As we write more code, we are going to write lots and lots of anonymous functions.
+
+JavaScript added even shorter ways of writing an anonymous function. These four functions all do the same thing:
```

Comment:
```suggestion
JavaScript has even shorter ways of writing an anonymous function. These four functions all do the same thing:
```

Reply:
I think you might be inadvertently telling them that arrow functions are anonymous functions here. Can you think of how to mitigate this risk?

Reply:
Technically they are, aren't they? An arrow function _is_ anonymous until it's bound to an identifier?

```js
// This function is anonymous, it just happens to be defined as an arrow function.
test("foo" () => {});

// This only stops being anonymous because it happens to be bound to an identifier.
// The fact that it's an arrow function is incidental.
const testFunction = () => {};
test("bar", testFunction);
```

Reply:
It is, but what I think could happen is that trainees who don't see a few examples of  `const namedFunction = () => ` could model this as arrow functions _are_ anonymous, as in this is a feature of arrow functions not of all functions.

Reply:
Added a clarifying blurb in aac5e1a1718f371bfb159c41526841eb77f8037f